src/gui/stac/qgsstacsourceselect.h

```diff
@@ -93,13 +97,19 @@ class GUI_EXPORT QgsStacSourceSelect : public QgsAbstractDataSourceWidget, priva
 
     void showItemsContextMenu( QPoint point );
 
+    void highlightFootprint( const QModelIndex &index );
+    void showFootprints( bool enable );
+    void loadUri( const QgsMimeDataUtils::Uri &uri );
+
     QString mCollectionsUrl;
     QString mSearchUrl;
     QUrl mNextPageUrl;
 
     QgsStacController *mStac = nullptr;
     QgsStacItemListModel *mItemsModel = nullptr;
     QgsStacSearchParametersDialog *mParametersDialog = nullptr;
+    std::unique_ptr<QgsRubberBand> mCurrentItemBand;
```

Comment:
Elsewhere we use:

```suggestion
    QObjectUniquePtr<QgsRubberBand> mCurrentItemBand;
```

so that there's no risk of double-deletion